bitnami/common/templates/_capabilities.tpl

```diff
@@ -84,6 +84,18 @@ Return the appropriate apiVersion for statefulset.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the appropriate apiVersion for job.
+*/}}
+{{- define "common.capabilities.job.apiVersion" -}}
+{{- $kubeVersion := include "common.capabilities.kubeVersion" . -}}
+{{- if and (not (empty $kubeVersion)) (semverCompare "<1.21-0" $kubeVersion) -}}
+{{- print "batch/v1beta1" -}}
+{{- else -}}
+{{- print "batch/v1" -}}
+{{- end -}}
+{{- end -}}
+
```

Comment:
I noticed the logic is exactly the same we have at `common.capabilities.cronjob.apiVersion`. I agree we could use different helpers in case in future K8s release Jobs & CronJobs API version differ (very unlikely but who knows 🤷🏼‍♂️ ), but please let's move this new function just before or after the existing one given they're related.

Reply:
Sorry, I didn't notice the cronjob template. I have updated.